sapi/cli/php_http_parser.h

```diff
@@ -79,6 +79,9 @@ enum php_http_method
   , PHP_HTTP_POST
   , PHP_HTTP_PUT
   , PHP_HTTP_PATCH
+  /* RFC-2068, section 19.6.1.2 */
```

Comment:
You're referring to RFC 2068 which has been obsoleted by RFC 2616 which has been obsoleted by RFC 7230ff.